config/locales/en.yml

```diff
@@ -817,7 +817,8 @@ en:
         deferral_notice_link_html: To use a different date, %{reinstatement_link}, then withdraw them.
       withdrawal_reasons:
         headings:
-          withdrawal_date: When did the trainee withdraw?
+          withdrawal_date: Date the trainee withdrew
+          withdrawal_date_hint: This is the last day they trained for %{award_type}
```

Comment:
In the prototype there's a full stop at the end of the hint.

<img width="463" alt="Screenshot 2023-04-17 at 15 04 40" src="https://user-images.githubusercontent.com/18436946/232479133-278bfcda-8f95-4c61-8afe-2c7f8fbddc35.png">